src/services/profiles/medications.ts

```diff
@@ -0,0 +1,68 @@
+import type { HttpInstance } from '../../types';
+import {
+  AffectedRecords,
+  ObjectId,
+  PagedResult,
+  Results,
+  ResultResponse,
+} from '../types';
+import { Medication } from './types';
+
+export default (client, httpAuth: HttpInstance) => ({
+  /**
+   * Retrieve a list of all the defined medications
+   * Permission | Scope | Effect
+   * - | - | -
+   * none | | Everyone can use this endpoint
+   *
+   * @returns PagedResult<Medication>
+   */
+  async getMedications(): Promise<PagedResult<Medication>> {
```

Comment:
Maybe `list` ?
It's an atypical PageResult as it just returns everything (138 strings) with no limit

Reply:
I was thinking about `list` but that means we need to change it everywhere (in the other services) so I left it as is for now, but we can ask in the call today, and then create an issue to update all services